doc/samtools-view.1

```diff
@@ -254,6 +256,20 @@ Note that records with no
 tag will also be output when using this option.
 This behaviour may change in a future release.
 .TP
+.BI "-d " STR:STR
+Only output alignments with tag
+.I STR
+and associated value
+.I STR
+[null].
+.TP
+.BI "-D " STR:FILE
```

Comment:
There is a minor issue with POSIX paths in MinGW environments, when using the colon (`:`) as a separator.
The colon is the POSIX path list delimiter and is [translated](http://www.mingw.org/wiki/Posix_path_conversion) by the MinGW CLI into a semi-colon (`;`), which is the Windows path list separator.
E.g. `-D BC:/c/tmpout/view.001.fobc view.001.sam` will be translated into `-D BC;C:\tmpout\view.001.fobc view.001.sam`, before being passed to samtools, which will result in an invalid option, as it stands.

That is why I still think a robust solution would be to have the tag inside the file, as the first line.

Reply:
@valeriuo I would prefer to not have the flag in the file as it makes generating the file a bit more cumbersome (else it is just a list of identifiers). It also wouldn't solve the problem for the `-d STR:STR` option.

What about `@` as delimiter?
`-d STR@STR` `-D STR@STR`

Reply:
Using colon in `BC:foo` has the advantage that pressing TAB in Unix shells will still do filename completion when you're typing _foo_. Ideally a workaround or alternative for MinGW can be come up with without pessimising that…

Reply:
I could add support for both (`:` and `@` delimiter).